examples/navier-stokes/navierstokes.c

```diff
@@ -356,7 +356,9 @@ int main(int argc, char **argv) {
   const CeedInt dim = 3, ncompx = 3, ncompq = 5;
   CeedVector xcorners, xceed, qdata, q0ceed, mceed, onesvec, multlvec;
   CeedBasis basisx, basisxc, basisq;
-  CeedTransposeMode lmodeceed = CEED_NOTRANSPOSE, lmodepetsc = CEED_TRANSPOSE;
+
+  CeedInterlaceMode imodeceed = CEED_NONINTERLACED,
+                    imodepetsc = CEED_INTERLACED;
```

Comment:
Sorry, I still wouldn't call these variables `imodeceed` and `imodepetsc`. It wouldn't say much to a new user who is learning from our examples as tutorials. What about something like `interlacenode` and `interlacecomp` (or `consecnode` and `conseccomp`)?

Reply:
I think that would make it more confusing - we decided interlaced referred to one specific meaning and our examples shouldn't muddle that single meaning.

I could directly type `CEED_INTERLACED` or `CEED_NONINTERLACED` into every one of those fields if its more clear.

Reply:
Updated